modules/data.remote/R/NLCD.R

```diff
@@ -104,14 +104,14 @@ extract_NLCD <- function(buffer, coords, data_dir = NULL, con = NULL, year = 201
         print(paste("File not found:", filename))
         return(NULL)
     }
-    nlcd <- raster(filename)
+    nlcd <- raster::raster(filename)
```

Comment:
Not part of this PR, but we should open a separate issue about switching all of our `raster` calls to `terra` since `raster` is deprecated

Reply:
I had the exact suspicion! raster is being used at a lot of places, so I'll leave it for now.

Reply:
Not to be pedantic, but `raster` has been *superseded*, not deprecated.  The difference is that `raster` isn't going away anytime soon. Still probably a good idea to do a spatial package makeover at some point!